airflow/configuration.py

```diff
@@ -311,7 +223,7 @@ def run_command(command):
 error_logfile = -
 
 # Expose the configuration file in the web server
-expose_config = true
+expose_config = false
```

Comment:
It actually doesn't matter -- in order to work with it users have to call getbool() (otherwise it's a string), which loads any capitalization. But I will make it False for clarity.